qiita_db/util.py

```diff
@@ -506,13 +506,15 @@ def compute_checksum(path):
     return crc & 0xffffffff
 
 
-def get_files_from_uploads_folders(study_id):
+def get_files_from_uploads_folders(study_id, ignore_hidden_and_folders=True):
```

Comment:
Not sure what you mean, wanna clarify? We could also discuss during
meeting, if easier.